course_discovery/apps/edly_discovery_app/api/v1/views/edly_sites.py

```diff
@@ -96,3 +95,24 @@ def get_updated_site_partner(self, discovery_site):
         partner.save()
 
         return partner
+
+
+class EdlySiteConfigViewset(APIView):
+    """
+    Create Default Site and Partner Configuration.
+    """
+    permission_classes = [IsAuthenticated, CanAccessSiteCreation]
+
+    def post(self, request):
+        """
+        POST /edly_api/v1/edly_site_config/
+        """
+        site = request.site
+        site_partner = Partner.objects.get(site=site)
+        request_data = request.data.get('discovery', {})
+        current_plan = request_data.get('DJANGO_SETTINGS_OVERRIDE', {}).get('CURRENT_PLAN', None)
```

Comment:
nit: `get()`of `dict` by default return None in case key is missing in dict.

Reply:
Done